infrastructure/lambda/template.yaml

```diff
@@ -228,16 +211,26 @@ Mappings:
       integration: MONTHS
       production: MONTHS
 
+  # Set as EnvVar on the CheckLambda
   # devEnvironmentOnlyEnhancedDebug changes CRI behaviour
   # Do not enable outside of Dev
-  devEnvironmentOnlyEnhancedDebugMapping:
+  DevEnvironmentOnlyEnhancedDebugMapping:
```

Comment:
Don't think these references are case sensitive but just double checking, is where this mapping is referenced updated aswell?

Reply:
Swapped to title case as part of the change to env var based feature flags.